.gitignore

```diff
@@ -5,4 +5,5 @@ assets/
 client/data
 client/imgui.ini
 levelpacker/FlatBuffGenerated/
-levels/*.bin
\ No newline at end of file
+levels/*.bin
+*.tsbin

```

Comment:
What are .tsbin files used for? From what I've seen tilesets are currently embedded in the level data and there is no need for separate files (as it should be)

Reply:
True, that's an uncleaned refactor artifact. Sry
```suggestion
```